packages/phase2cli/src/lib/auth.ts

```diff
@@ -0,0 +1,106 @@
+import Conf from "conf"
+import { FirebaseApp } from "firebase/app"
+import { signInToFirebaseWithGithubToken, getCurrentFirebaseAuthUser } from "@zkmpc/actions"
+import { IdTokenResult, User } from "firebase/auth"
+import { AuthUser } from "../../types/index"
+import { readLocalJsonFile } from "./files"
+import { GENERIC_ERRORS, GITHUB_ERRORS, showError } from "./errors"
+import { emojis, theme } from "./constants"
+import { getGithubUsername } from "./utils"
+
+// Get local configs.
+const { name } = readLocalJsonFile("../../package.json")
+
+// Local configstore for storing auth data (e.g., tokens).
+const config = new Conf({
+    projectName: name,
```

Comment:
What does this define? The name of the project on firebase?

Reply:
My apologies for the lack of clarity here. This code creates a new local storage (named as the name of the project/package) where the Github token resulting from the handshake performed during the `auth` CLI command is stored (user local machine only). Then, it will be possible to use this token to verify and/or refresh user authentication.

FYI: Currently, there is this issue open #171 to complete the expiration/refresh feature.